### PR TITLE
Included libselinux-python dependency

### PR DESCRIPTION
**Pull request by jbadiapa**
On a server installed only with the base the libselinux-python module is not installed by default

### CODE REVIEW

File: roles/dependencies/libselinuxpython/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+
+- name: install libselinux python
+  yum:
+      name: '{{ libselinux_python_package_name }}'
+      state: present
```
Comment (larsks) [inline roles/dependencies/libselinuxpython/tasks/main.yml]: `when: manage_packages|default(false)`

**Review by larsks — CHANGES_REQUESTED** [review]
I agree that we need this.  How about we move this to `roles/prereqs/libselinuxpython`, and then create `roles/prereqs/meta/main.yml` with:

```
---
dependencies:
  - prereqs/libselinuxpython
```

And then make all other roles depend on the `prereqs` role?

File: roles/dependencies/libselinuxpython/tasks/main.yml (hunk 1 above)
Comment (jbadiapa) [reply roles/dependencies/libselinuxpython/tasks/main.yml]: yes, I forgot this

**Comment by jbadiapa** [comment]
Ok

File: roles/prereqs/libselinuxpython/tasks/main.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 
 - name: install libselinux python
-  yum:
+  package:
```
Comment (jbadiapa) [inline roles/prereqs/libselinuxpython/tasks/main.yml:4]: I need to review the way of manage packages I think I used yum instead of package

**Pull request merged**